.github/workflows/tests_and_coverage.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   tests-coverage:
-    runs-on: ubuntu-latest
+    runs-on: macos-13
```

Comment:
I think we want this as close to hasta and the vms as possible, i.e. CentOS(?) or similar.

Reply:
Changed it to ubuntu, there are no runners with centos